frontend/src/ui/Blocks/VideoList.tsx

```diff
@@ -109,6 +111,7 @@ export const VideoListBlock: React.FC<VideoListBlockProps> = ({
 }) => {
     const { t, i18n } = useTranslation();
     const [eventOrder, setEventOrder] = useState<Order>(initialOrder);
+    const user = useUser() !== "none";
```

Comment:
I would do `const user = useUser();` here and below do `isRealUser(user)`. That seems more intuitive to me.